### PR TITLE
[DO NOT MERGE] Enable Chartbeat toggle on LIVE

### LINKED ISSUE #2806: Enable Chartbeat toggle on live
**Is your feature request related to a problem? Please describe.**
Once Chartbeat is working properly on test, we should flip the Chartbeat toggle to enable it on live.

**Describe the solution you'd like**
Change this boolean to `true` to enable Chartbeat analytics on live.
https://github.com/bbc/simorgh/blob/fce62cd9cc63cbe5af0147d258f30e9073acaa08/src/app/lib/config/toggles/index.js#L20-L22

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**DEPENDS ON** 
Chartbeat testing https://github.com/bbc/simorgh/issues/2808 

**Additional context**

### PR DESCRIPTION
**Pull request by 12**
Resolves #2806 

**Overall change:** Enable Chartbeat on LIVE. Blocked on #2811. 

**Code changes:**

- Change Chartbeat toggle to `true` for `live` environment
- Update snapshots

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Comment by 12** [comment]
This is not to be merged until #2811 is merged.

**Review by jamesdonoh — APPROVED** [review]

File: src/app/contexts/ToggleContext/__snapshots__/index.test.jsx.snap
Comment (chris-hinds) [inline src/app/contexts/ToggleContext/__snapshots__/index.test.jsx.snap:5]: Nothing wrong with this PR, but maybe me implementing a snapshot test for this was not the best idea. means this snapshot will change and need to be updated everytime we change a toggle.

**Review by chris-hinds — APPROVED** [review]

File: src/app/contexts/ToggleContext/__snapshots__/index.test.jsx.snap
Comment (jamesdonoh) [reply src/app/contexts/ToggleContext/__snapshots__/index.test.jsx.snap:5]: Yeah, looks like the toggles code needs some more explicit tests. For the present would it be worth at least mocking out `lib/config/toggles` in the `ToggleContext` unit test with some dummy keys/values? At least then the snapshot won't keep needing to be updated when toggles change.

Comment (chris-hinds) [reply src/app/contexts/ToggleContext/__snapshots__/index.test.jsx.snap:5]: That should work @jamesdonoh . I'll raise another issue to do that later.

**Comment by 12** [comment]
This is blocked on #3126. Chartbeat will go live once we are satisfied the configuration option is itself configured correctly.

**Comment by jroebu14** [comment]
closing as work was done in https://github.com/bbc/simorgh/pull/4894

**Pull request closed without merging**
